webpack.config.js

```diff
@@ -60,17 +43,15 @@ const config = {
   },
   plugins: [
     new CopyPlugin({
-      patterns: [{ from: "assets", to: "assets" }],
+      patterns: [
+        { from: "node_modules/@biz-dev-ops/md-docs/assets", to: "assets" },
```

Comment:
This needs to be added for dev build?

Reply:
@jessevanmuijden this file loads the css files, must be removed? https://github.com/biz-dev-ops/web-components/blob/fix/build/website/src/index.ts

Reply:
no, webpack.config.dev.js is for the dev build. removing that index.ts file will break the dev website, but I can try to wrap the imports in an if-statement.